packages/x-charts-pro/src/internals/plugins/useChartProZoom/ZoomInteractionConfig.test.tsx

```diff
@@ -363,4 +363,173 @@ describe.skipIf(isJSDOM)('ZoomInteractionConfig Keys and Modes', () => {
       expect(getAxisTickValues('x').length).to.be.lessThan(4);
     });
   });
+
+  describe('Zoom and Pan with reversed axis', () => {
+    it('should zoom at the correct position with reversed x-axis', async () => {
+      const onZoomChange = sinon.spy();
+      const { user } = render(
+        <BarChartPro
+          series={[
+            {
+              data: [10, 20, 30, 40],
+            },
+          ]}
+          xAxis={[
+            {
+              data: ['A', 'B', 'C', 'D'],
+              zoom: true,
+              height: 30,
+              id: 'x',
+              reverse: true, // Reversed axis
+            },
+          ]}
+          yAxis={[{ position: 'none' }]}
+          width={100}
+          height={130}
+          margin={0}
+          slotProps={{ tooltip: { trigger: 'none' } }}
+          onZoomChange={onZoomChange}
+        />,
+        options,
+      );
+
+      // The reverse property affects scale mapping, not tick order
+      // Ticks still render in data order: A, B, C, D
+      expect(getAxisTickValues('x')).to.deep.equal(['A', 'B', 'C', 'D']);
+
+      const svg = document.querySelector(CHART_SELECTOR)!;
+
+      await user.pointer([
+        {
+          target: svg,
+          coords: { x: 25, y: 50 }, // Point near left edge
+        },
+      ]);
+
+      // Zoom in with a larger delta (similar to other zoom tests)
+      fireEvent.wheel(svg, { deltaY: -500, clientX: 25, clientY: 50 });
+      await act(async () => new Promise((r) => requestAnimationFrame(r)));
+
+      expect(onZoomChange.callCount).to.be.greaterThan(0);
+
+      // After zooming, should have fewer ticks than before
+      const ticksAfterZoom = getAxisTickValues('x');
+      expect(ticksAfterZoom.length).to.be.lessThan(4);
+      expect(ticksAfterZoom.length).to.be.greaterThan(0);
```

Comment:
This test isn't very robust. Shouldn't we be asserting that 'D' and maybe 'C' are visible? 

Otherwise if zoom is zooming into the wrong set of ticks this test would still pass, right?